core/src/main/resources/error/error-classes.json

```diff
@@ -52,17 +101,44 @@
     "message" : [ "PARTITION clause cannot contain a non-partition column name: %s" ],
     "sqlState" : "42000"
   },
+  "PARTITION_COLUMN_NOT_FOUND_IN_SCHEMA_ERROR" : {
+    "message" : [ "Partition column %s not found in schema %s" ]
+  },
   "PIVOT_VALUE_DATA_TYPE_MISMATCH" : {
     "message" : [ "Invalid pivot value '%s': value data type %s does not match pivot column data type %s" ],
     "sqlState" : "42000"
   },
+  "RENAME_AS_EXISTS_PATH_ERROR" : {
```

Comment:
This doesn't make much sense grammatically; maybe `RENAME_OVERWRITES_EXISTING_PATH`? We can also consolidate this with `RENAME_PATH_AS_EXISTS_PATH_ERROR`, and have everyone throw this error class.